BSP/Inc/BSP_ADC.h

```diff
@@ -0,0 +1,82 @@
+#ifndef __BSP_ADC_H
+#define __BSP_ADC_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "common.h"
+#include "config.h"
+
+enum ADCHardware {Accelerator, Brake};
+
+/**
+ * @brief   Initialize the ADC module
+ * @param   None
+ * @return  None
+ */ 
+void BSP_ADC_Init(void);
+
+/**
+ * @brief   Provides the number of active ADC channels 
+ * @param   None
+ * @return  numberOfADCValues, the number of ADC channels connected to the ADC providing raw ADC data
+ */ 
+int BSP_ADC_Number_Values();
+
+/**
+ * @brief   Provides pointer to array with the ADC values of csv file
+ * @param   None
+ * @return  rawADCValues, pointer to array that contains the ADC values in the array
+ */ 
+int* BSP_ADC_Read_Values();
+
+/**
+ * @brief   Provides the ADC value of the channel at the specified index
+ * @param   indexOfValue, index of value in the array of all ADC values obtained with read_ADC_Values
+ * @return  rawADCValue
+ */ 
+int BSP_ADC_Get_Value(int indexOfValue);
+
+/**
+ * @brief   Provides the ADC value in millivolts of the channel at the specified index
+ * @param   indexOfValue, index of value in the array of all ADC values obtained with read_ADC_Values
+ * @return  ADC value in millivolts
+ */ 
+float BSP_ADC_Get_Millivoltage(int indexOfValue);
```

Comment:
I also want to keep this function. However, we want to avoid using floating point numbers. Since we are returning a millivoltage, you should just be able to return the integer portion of that number, since it will be precise to 3 decimal places as a voltage. 

Also, same as above for the paramater